gwcli/ceph.py

```diff
@@ -76,14 +76,20 @@ def get_clusters(self):
             name = os.path.basename(conf).split('.')[0]
             keyring = glob.glob(os.path.join(CephGroup.ceph_config_dir,
```

Comment:
I think this can be yanked as well

Reply:
I was using this to infer the client name. Are you suggesting to remove all lines below down to `local = True if name ...`?

Reply:
TBH, I don't understand the purpose of this code (detect several clusters)? Are there plans for ceph-iscsi to use RBD images from different clusters?

Reply:
To be honest, I don't understand it either. The backing API would only work with a single cluster specified in the config. It might have been a partially implemented thing for supporting mirroring, but honestly I think it can all be removed since it should be getting the mirror peer data from the pool.

Reply:
In that case I'll remove that code in this PR.